.github/workflows/build.yml

```
@@ -139,6 +139,12 @@ jobs:
       if: runner.os == 'Windows'
       uses: seanmiddleditch/gha-setup-vsdevenv@v3
 
+    - name: "[macOS] install ccache and make"
+      if: runner.os == 'macOS'
+      run: |
+        brew install ccache
+        brew install ninja
```

```suggestion
        brew install ccache ninja
```
can be combined to one command